cli/src/main/java/de/jplag/cli/CliOptions.java

```diff
@@ -53,6 +53,9 @@ public class CliOptions implements Runnable {
             "--result-directory"}, description = "Name of the directory in which the comparison results will be stored (default: ${DEFAULT-VALUE})%n")
     public String resultFolder = "results";
 
+    @Option(names = {"--mode"}, description = "The mode to run jplag in")
```

Comment:
Two issues here:

- the default value is missing
- this is very much a central new feature, but the description is lacking.

```suggestion
    @Option(names = {"--mode"}, description = "The mode of JPlag: either only run analysis, only open the viewer, or do both (default: ${DEFAULT-VALUE})")
```